### PR TITLE
Feature Branch finished

### PR DESCRIPTION
**Pull request by diazgio**
We finish the project already, We're waiting for the review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+![screenshot](./images/screen-shot.PNG)
+
+
+## Built With
+
+- HTML and CSS3
+
+## Live Demo
+
+[Live Demo Link](https://diazgio.github.io/Mint-singup-project/)
```
Comment (BrittanyBlake) [inline README.md:14]: The live demo link for me was not working ☹️ Please try to fix this.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+}
+
+.terms p {
+    color: rgb(69, 69, 69);
+}
+
+.terms a {
+    color: #0077C5;
+    text-decoration: none;
+}
```
Comment (BrittanyBlake) [inline style.css:158]: This code can be grouped with the code in **lines 174 - 177** as they do the same thing.

**Comment by Miguelus373** [comment]
We remade the main logo with HTML and CSS

**Comment by Miguelus373** [comment]
We added all requests from the reviewer

File: style.css [hunk 3]
```diff
@@ -167,4 +167,23 @@ body {
   font-size: 10px;
   text-align: center;
   color: rgb(69, 69, 69);
+}
+
+.logo {
+    display: flex;
```
Comment (bertil291utn) [inline style.css]: ## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Comment (diazgio) [reply style.css]: We change the classes and as you requested and fixing the padding too, also we change the main section to <main> tag.

**Pull request merged**